### ISSUE TITLE
Problems with firstDay from moment.js locale

### ISSUE THREAD

**Issue by CasperWSchmidt** [dangrossman/bootstrap-daterangepicker#401]
Hi

I just started using this nice tool along with moment.js. After fiddling around with moment to use localization I discovered an issue with the firstDay.
First a bit of sample code:
```javascript
moment.lang('da');
$("#inpFromDate").daterangepicker({
    format: 'DD-MM-YYYY',
    ranges: {
        '<% =JT("Today") %>': [moment(), moment()],
        '<% =JT("Yesterday") %>': [moment().subtract('days', 1), moment().subtract('days', 1)],
        '<% =Replace(JT("Last %1 Days"), "%1", 7) %>': [moment().subtract('days', 6), moment()],
        '<% =Replace(JT("Last %1 Days"), "%1", 30) %>': [moment().subtract('days', 29), moment()],
        '<% =JT("This Month") %>': [moment().startOf('month'), moment().endOf('month')],
        '<% =JT("Last Month") %>': [moment().subtract('month', 1).startOf('month'), moment().subtract('month', 1).endOf('month')]
    },
    startDate: '<% =objFilter.DateFrom %>',
    endDate: '<% =objFilter.DateTo %>',
    maxDate: moment().endOf("month"),
    locale: {
        applyLabel: '<% =JT("Apply") %>',
        cancelLabel: '<% =JT("Cancel") %>',
        fromLabel: '<% =JT("From") %>',
        toLabel: '<% =JT("To") %>',
        weekLabel: '<% =JT("W") %>',
        customRangeLabel: '<% =JT("Custom Range") %>'
   }
},
function (start, end) {
    $('#inpFromDate span').html(start.format('DD-MM-YYYY') + ' - ' + end.format('DD-MM-YYYY'));
}
```
![image](https://cloud.githubusercontent.com/assets/3481853/3763815/84e0adb4-18a8-11e4-8ff3-64a18c61aa63.png)
According to line 164 (firstDay: moment()._lang._week.dow) the firstDay parameter should be set correcly, but looking at the screenshot it is not. Using console.log(moment()._lang._week.dow); correctly shows that dow is 1. 

Here comes the funny thing. If I modify my daterangepicker options to include firstDay: moment()._lang._week.dow so that my locale looks like this:
```javascript
locale: {
    applyLabel: '<% =JT("Apply") %>',
    cancelLabel: '<% =JT("Cancel") %>',
    fromLabel: '<% =JT("From") %>',
    toLabel: '<% =JT("To") %>',
    weekLabel: '<% =JT("W") %>',
    customRangeLabel: '<% =JT("Custom Range") %>',
    firstDay: moment()._lang._week.dow
}
```
Everything is fine:
![image](https://cloud.githubusercontent.com/assets/3481853/3763461/00dc16de-18a3-11e4-9b69-10c87b123dfd.png)

Can someone please help me out?

oh and for the record. <% =JT("sometext") %> is an ASP Classic (VBScript) function that first translates the text and then makes it javascript-safe (escaping certain characters). <% =objFilter.DateFrom %> and <% =objFilter.DateTo %> returns dates as strings (fx. "01-07-2014" and "31-07-2014")

**Comment by CasperWSchmidt** [comment]
Was this issue fixed or just closed as "wont fix"?

**Comment by dangrossman** [comment]
I closed all outstanding issues against v1 when releasing v2. I honestly don't know if this bug still exists.

**Comment by CasperWSchmidt** [comment]
The issue I filed against moment was fixed back in October for v2.9. If moment was updated when you released v2 it should be easy to fix (see https://github.com/moment/moment/pull/2128/files). Unfortunately I don't have much time to review all your the source code right now :(

**Comment by dangrossman** [comment]
Thanks for the pointer. The day of week was already being read from the localeData() here too, so it probably was fixed a while ago.
